### ISSUE TITLE
Why crconsole would not exit when I redirect the stdin with an input file ?

### ISSUE THREAD

**Issue by zyscoder** [sidorares/crconsole#34]
I am sorry to bother you. I try to redirect stdin with an input file to send my javascript code to crconsole, and I am confused why the process of crconsole would not exit after it received all the input. Would you mind giving me some advice about this problem if you have any idea?

**Comment by sidorares** [comment]
you probably need to close stdin ( just a guess ).

It's been a while I had a look at this project, can you a bit of context why are you redirecting stdin and what's expected result?

**Comment by zyscoder** [comment]
This is my context. The process of crconsole cannot exit otherwise I a give it another ^C.
![2020-07-06 19-27-27 的屏幕截图](https://user-images.githubusercontent.com/23655282/86588705-cbac9c80-bfbe-11ea-9dba-20a445354493.png)
My expect of it is like the behavior of nodejs, who can exit the process nomally after executing all the code.
![2020-07-06 19-29-54 的屏幕截图](https://user-images.githubusercontent.com/23655282/86589053-7ae97380-bfbf-11ea-9ba1-5b837ac38f83.png)

**Comment by sidorares** [comment]
Yeah, at the moment it always starts in repl mode, I can see that "execute and exit" might be useful

**Comment by zyscoder** [comment]
Is there any way to achieve the "execute and exit"? Maybe I can have a try.
